nomad/secure_variables_endpoint.go

```diff
@@ -464,3 +497,53 @@ func (sv *SecureVariables) authValidatePrefix(claims *structs.IdentityClaims, ns
 	}
 	return nil
 }
+
+func (s *SecureVariables) validateCASUpdate(cidx uint64, sv *structs.SecureVariableDecrypted, conflict **structs.SecureVariableDecrypted) error {
+	return s.validateCAS(cidx, sv.Namespace, sv.Path, conflict)
+}
+
+func (s *SecureVariables) validateCASDelete(cidx uint64, namespace, path string, conflict **structs.SecureVariableDecrypted) error {
+	return s.validateCAS(cidx, namespace, path, conflict)
+}
+
+func (s *SecureVariables) validateCAS(cidx uint64, namespace, path string, conflictOut **structs.SecureVariableDecrypted) error {
+	casConflict := errors.New("conflict")
```

Comment:
For later: maybe we can move this error to `nomad/structs/errors.go`. Then we can use nicer constructs like `errors.Is` within the RPC handler instead of `if err.Error() == "conflict"` (which we can't avoid with the HTTP handler unfortunately).